apps/alert_processor/test/alert_processor/dissemination/mass_notifier_test.exs

```diff
@@ -62,10 +62,23 @@ defmodule AlertProcessor.Dissemination.MassNotifierTest do
     end
 
     test "enqueues notifications with a phone number ahead of others" do
-      {_, first_phone} = insert_user_and_build_notification(%{phone_number: "5555551234"})
-      {_, first_email} = insert_user_and_build_notification(%{phone_number: nil})
-      {_, second_phone} = insert_user_and_build_notification(%{phone_number: "5555556789"})
-      {_, second_email} = insert_user_and_build_notification(%{phone_number: nil})
+      {_, first_phone} =
+        insert_user_and_build_notification(%{
+          communication_mode: "sms",
+          phone_number: "5555551234"
+        })
+
+      {_, first_email} =
+        insert_user_and_build_notification(%{communication_mode: "email", phone_number: nil})
```

Comment:
suggestion (non-blocking): It was a bit confusing to me as someone not familiar with this codebase why phone number was being set explicitly but email wasn't. I see now that it is set by default from the user factory, so no change really needed here. Maybe for consistency since the phone_number is explicitly set for the `first_phone` and `second_phone` users, consider setting the email for `first_email` and `second_email` users too.